include/zm/zm_pex.h

```diff
@@ -44,8 +44,13 @@ typedef zVec zPex;
 #define zPexClone(p)             zVecClone(p)
 #define zPexCloneArray(a,n)      zVecCloneArray(a,(n)+1)
 
+#ifdef __WINDOWS__
```

Comment:
Do not use platform-dependent macros in other files than zeda/include/zeda/zeda_compat.h.